package/MDAnalysis/analysis/base.py

```diff
@@ -124,6 +124,12 @@ def _setup_frames(self, trajectory, start=None, stop=None, step=None):
             stop frame of analysis
         step : int, optional
             number of frames to skip between each analysed frame
+        verbose : bool, optional
+            Turn on verbosity
+            
+        .. versionchanged:: 1.0.0
+            Verbose argument is now passed explicitly from
+            :meth:`AnalysisBase.run`.
         """
         self._trajectory = trajectory
         start, stop, step = trajectory.check_slice_indices(start, stop, step)
```

Comment:
Since this is historical AnalysisBase stuff, @richardjgowers might be best placed to discuss if there were any specific reasons for this (I think it was added in #460).

This behaviour is probably going to need to change in light of #2206, so I'm not sure if it's not best to just tackle all this in a separate PR?

Reply:
These are valid comments, though these changes are out of scope of this pull request.